From fedd3e5275d3388e8d213c592aeca50640ceb372 Mon Sep 17 00:00:00 2001
From: Daniele Tricoli <eriol@mornie.org>
Date: Fri, 23 Oct 2015 16:03:52 +0200
Subject: Populate install_requires for unbundled packages

This will avoid breakage updating urllib3 via pip when requests/urllib3
are already installed via the system packages.

Forwarded: not-needed
Bug-OpenStack: https://review.openstack.org/#/c/213310/0
Bug-Upstream: https://github.com/kennethreitz/requests/issues/2816
Patch-Name: 02_populate-install_requires.patch
---
 setup.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c483535..2648a3a 100755
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,9 @@ packages = [
     'requests.packages.urllib3.packages.ssl_match_hostname',
 ]
 
-requires = []
+requires = [
+    'urllib3==1.13.1',
+]
 
 version = ''
 with open('requests/__init__.py', 'r') as fd:
